recipes/nghttp2/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "nghttp2" %}
+{% set version = "1.39.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nghttp2/{{ name }}/releases/download/v{{version}}/nghttp2-{{ version }}.tar.gz
+  sha256: 25b623cd04dc6a863ca3b34ed6247844effe1aa5458229590b3f56a6d53cd692
+
+build:
+  number: 0
+  skip: True  # [not unix]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - pkg-config  # [unix]
+    - zlib>=1.2.3
```

Comment:
These 4 should be in `host only` (only build tools like the compilers and make belong in `build`).